charts/admission-controller/templates/_helpers.tpl

```diff
@@ -395,6 +395,7 @@ webhooks:
     apiVersions: ["v1"]
     operations: ["CREATE", "UPDATE"]
     resources:
+    - "pods"
```

Comment:
do we need to wrap this with the conditional too? Or is it ok to let KSPM just ignore the pods? (They should be eventually supported in the future)

Reply:
IIRC (@dimar-apolicy correct me if I'm wrong) KSPM will ignore the pods events as of now.

I can put a guard on that too, though

Reply:
done @airadier